openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx

```diff
@@ -159,10 +159,10 @@ const ActivityFeedList: FC<ActivityFeedListProp> = ({
               tag="button"
               variant="link"
               onClick={() => setFieldListVisible((visible) => !visible)}>
-              <span className="tw-font-medium tw-text-grey">
+              <span className="tw-font-medium tw-text-primary">
                 {feedFilterList.find((f) => f.value === feedFilter)?.name}
               </span>
-              <DropDownIcon />
+              <DropDownIcon style={{ color: '#7147E8' }} />
```

Comment:
Can we avoid this inline styling?

Reply:
@Sachin-chaurasiya  This dropdown icon is a generic component and default color is black, so as per our need for our component we have to pass it.